### PR TITLE
[WIP]Defining path in e2e suite for dev console tests

### PR DESCRIPTION
**Pull request by sanketpathak**

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -90,7 +90,8 @@ const testSuites = {
     'tests/monitoring.scenario.ts',
     'tests/alertmanager.scenario.ts',
     'tests/crd-extensions.scenario.ts',
-    'tests/devconsole/pipeline.scenario.ts',
+    '../packages/dev-console/integration-tests/tests/dev-perspective.scenario.ts',
+    '../packages/dev-console/integration-tests/tests/pipeline.scenario.ts',
```
Comment (spadgett) [inline frontend/integration-tests/protractor.conf.ts]: This shouldn't be necessary. The plugin can add its tests to e2e in package.json. Here's an example:

https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/package.json#L19-L21

File: frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { browser, $, ExpectedConditions as until, by, element } from 'protractor';
 
-export const operatorModal = $('.pf-c-modal-box');
+export const operatorModal = $('.co-catalog-page__overlay');
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts]: Is this meant to be part of this PR?

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (sanketpathak) [reply frontend/integration-tests/protractor.conf.ts]: Will this allow the above tests to run on every pr

File: frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts (hunk 2 above)
Comment (sanketpathak) [reply frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts]: Yes when I was running tests (regarding installing pipeline operator) it was failing so I pushed this change along with this pr

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (spadgett) [reply frontend/integration-tests/protractor.conf.ts]: Yes, if you add to the e2e suite in this way, the tests will run against PRs and gate merge.

File: frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts (hunk 2 above)
Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts]: We should consider a `data-test-id` here, which would be less fragile.

Comment (sanketpathak) [reply frontend/packages/operator-lifecycle-manager/integration-tests/views/operator-hub.view.ts]: I agree with your point, but for this, I want help from someone in openshift dev team on this.
About your first point regarding `Is this meant to be part of this pr` I've edited the pr remove pipeline test from e2e suite for now and included the above `operatorModal` change in my pipeline specific test pr #4147

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (sanketpathak) [reply frontend/integration-tests/protractor.conf.ts]: Acknowledged, I'll edit my pr.

**Comment by sanketpathak** [comment]
Not working on this right now so closing it, will reopen again later

**Pull request closed without merging**